supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/HtCondorJobExecutionActor.scala

```diff
@@ -269,7 +269,9 @@ class HtCondorJobExecutionActor(override val jobDescriptor: BackendJobDescriptor
       executionDir.toString.toFile.createIfNotExists(asDirectory = true, createParents = true)
 
       log.debug("{} Resolving job command", tag)
-      val command = localizeInputs(jobPaths.callRoot, runtimeAttributes.dockerImage.isDefined, fileSystems, jobDescriptor.inputs) flatMap {
+      val allowSoftLinks = configurationDescriptor.backendConfig.getBoolean("docker.allowSoftLinks")
+      val dockerLocEnabled = runtimeAttributes.dockerImage.isDefined && !allowSoftLinks
+      val command = localizeInputs(jobPaths.callRoot, dockerLocEnabled, fileSystems, jobDescriptor.inputs) flatMap {
```

Comment:
By tomorrow I should have an answer if I need to proceed with this change or not. 
The idea is to merge configurable docker command and try to avoid the merge of allowSoftLinks.